library/crates/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2021 Vaticle
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+# Script for regenerating BUILD files after Cargo.toml update
+
+set -ex
+
+[[ $(readlink $0) ]] && path=$(readlink $0) || path=$0
+CARGO_TOML_HOME=$(cd "$(dirname "${path}")" && pwd -P)
+
+pushd "$CARGO_TOML_HOME" > /dev/null
```

Comment:
I think the variable name should be `HOME`, because we're setting the "home directory" for this script right?

Reply:
@lolski in UNIX systems, `HOME` is a reserved name with a well-understood meaning "home directory of the user". 

What we're really setting here is the _current_ directory for `cargo raze` to operate in - which has to be same directory as the one `Cargo.toml` resides in.

For reference, in `bazel-deps`'s `update.sh` it was called `GRAKN_CORE_HOME` because it pointed to the root of the git repo.

Reply:
@lolski addressed in https://github.com/vaticle/dependencies/pull/318/commits/94a192c1602db679d8ec1162e4906539f9f2f08b